pkg/login/social/github_oauth.go

```diff
@@ -67,7 +68,7 @@ func (s *SocialGithub) IsOrganizationMember(client *http.Client, organizationsUr
 
 	for _, allowedOrganization := range s.allowedOrganizations {
 		for _, organization := range organizations {
-			if organization == allowedOrganization {
+			if strings.ToLower(organization) == strings.ToLower(allowedOrganization) {
```

Comment:
I have just seen the linter error. Could you please change this part to use the `strings.EqualFold()` function?

Reply:
Ah, interesting! Done in 1d221fe!